content/publications/_index.md

```diff
@@ -2,6 +2,9 @@
 bibliography:
 - title: 2020
   items:
+  - yoshitom-vu-2020-sigir
+  - yoshitom-levorato-2020-emdl
+  - yoshitom-levorato-2020-icpr
```

Comment:
These two entries (using their corrected IDs) should also be added to
Marco’s bibliography in `content/people/levorato.md`.